Source/Core/VideoBackends/D3D12/D3DQueuedCommandList.cpp

```diff
@@ -0,0 +1,1213 @@
+						src_box->top == 0;
+
+					command_list->CopyTextureRegion(
+						&reinterpret_cast<D3DQueueItem*>(item)->CopyTextureRegion.dst,
+						0, // item->CopyTextureRegion.DstX, (always zero, checked at command enqueing time)
+						0, // item->CopyTextureRegion.DstY, (always zero, checked at command enqueing time)
+						0, // item->CopyTextureRegion.DstZ, (always zero, checked at command enqueing time)
+						&reinterpret_cast<D3DQueueItem*>(item)->CopyTextureRegion.src,
+						empty_box ?
+							nullptr : src_box
+						);
+
+					item += sizeof(CopyTextureRegionArguments) + sizeof(D3DQueueItemType) * 2;
+					break;
+				}
+
+				case D3DQueueItemType::DrawIndexedInstanced:
+				{
+					command_list->DrawIndexedInstanced(
+						reinterpret_cast<D3DQueueItem*>(item)->DrawIndexedInstanced.IndexCount,
+						1,
+						reinterpret_cast<D3DQueueItem*>(item)->DrawIndexedInstanced.StartIndexLocation,
+						reinterpret_cast<D3DQueueItem*>(item)->DrawIndexedInstanced.BaseVertexLocation,
+						0
+						);
+
+					item += sizeof(DrawIndexedInstancedArguments) + sizeof(D3DQueueItemType) * 2;
+					break;
+				}
+
+				case D3DQueueItemType::DrawInstanced:
+				{
+					command_list->DrawInstanced(
+						reinterpret_cast<D3DQueueItem*>(item)->DrawInstanced.VertexCount,
+						1,
+						reinterpret_cast<D3DQueueItem*>(item)->DrawInstanced.StartVertexLocation,
+						0
+						);
+
+					item += sizeof(DrawInstancedArguments) + sizeof(D3DQueueItemType) * 2;
+					break;
+				}
+
+				case D3DQueueItemType::IASetPrimitiveTopology:
+				{
+					command_list->IASetPrimitiveTopology(reinterpret_cast<D3DQueueItem*>(item)->IASetPrimitiveTopology.PrimitiveTopology);
+
+					item += sizeof(IASetPrimitiveTopologyArguments) + sizeof(D3DQueueItemType) * 2;
+					break;
+				}
+
+				case D3DQueueItemType::ResourceBarrier:
+				{
+					command_list->ResourceBarrier(1, &reinterpret_cast<D3DQueueItem*>(item)->ResourceBarrier.barrier);
+
+					item += sizeof(ResourceBarrierArguments) + sizeof(D3DQueueItemType) * 2;
+					break;
+				}
+
+				case D3DQueueItemType::RSSetScissorRects:
+				{
+					D3D12_RECT rect = {
+						reinterpret_cast<D3DQueueItem*>(item)->RSSetScissorRects.left,
+						reinterpret_cast<D3DQueueItem*>(item)->RSSetScissorRects.top,
+						reinterpret_cast<D3DQueueItem*>(item)->RSSetScissorRects.right,
+						reinterpret_cast<D3DQueueItem*>(item)->RSSetScissorRects.bottom
+					};
+
+					command_list->RSSetScissorRects(1, &rect);
+					item += sizeof(RSSetScissorRectsArguments) + sizeof(D3DQueueItemType) * 2;
+					break;
+				}
+
+				case D3DQueueItemType::RSSetViewports:
+				{
+					D3D12_VIEWPORT viewport = {
+						reinterpret_cast<D3DQueueItem*>(item)->RSSetViewports.TopLeftX,
+						reinterpret_cast<D3DQueueItem*>(item)->RSSetViewports.TopLeftY,
+						reinterpret_cast<D3DQueueItem*>(item)->RSSetViewports.Width,
+						reinterpret_cast<D3DQueueItem*>(item)->RSSetViewports.Height,
+						D3D11_MIN_DEPTH,
+						D3D11_MAX_DEPTH

```

Comment:
you have to add support for min and max depth value, as they are modified in Render.cpp this will fix a lot of games.